mmdet3d/models/dense_heads/centerpoint_head.py

```diff
@@ -78,15 +80,18 @@ def __init__(self,
 
             self.__setattr__(head, conv_layers)
 
+            if init_cfg is None:
+                self.init_cfg = dict(
+                    type='Kaiming',
+                    layer='Conv2d',
+                    override=dict(type='Kaiming', name='heatmap'))
```

Comment:
I think override can be deleted, as you have init ``heatmap`` in init_weights.

Reply:
> Yes, I forget to delete it. thank you!